docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,272 @@
+./src/k8s/tools/regctl.sh USERNAME="$username" PASSWORD="$password" MIRROR="$mirror"
+```
+
+#### Images Option C: Side-load images
+
+Image side-loading is the process of loading all required OCI images directly
+into the container runtime, so that they do not have to be fetched at runtime.
+
+To create a bundle of images, you can use the [regctl][regctl] tool.
+
+```bash
+regctl image export --platform=local
+```
+
+Upon choosing this option, you place all images under
+`/var/snap/k8s/common/images` and they will be picked up by containerd.
+
+## Deploy Canonical Kubernetes
+
+Now that you have fulfilled all steps in preparation for your
+air gapped cluster, it is time to get it deployed.
+
+### Step 1: Install Canonical Kubernetes
+
+Copy the `k8s.snap`, `k8s.assert`, `core20.snap` and `core20.assert` files into
+the target node, then install the k8s snap by running:
+
+```bash
+sudo snap ack core20.assert && sudo snap install ./core20.snap
+sudo snap ack k8s.assert && sudo snap install ./k8s.snap --classic
+```
+
+Repeat the above for all nodes of the cluster.
+
+### Step 3: Container Runtime
+
+The container runtime needs to be configured to be ble to fetch images.
+
+#### Container Runtime Option A: Configure HTTP proxy for registries
+
+Edit `/etc/systemd/system/snap.k8s.containerd.conf.d/env.conf`
+and set the appropriate http_proxy, https_proxy and
+no_proxy variables as described in the
+[adding proxy configuration section][proxy].
+
+#### Container Runtime Option B: Configure registry mirrors
+
+This requires that you have already setup a registry mirror,
+as explained in the preparation section on the private registry mirror.
+For each upstream registry that you want to mirror, create a hosts.toml file.
+
+This example configured `http://10.100.100.100:5000` as a mirror for
+`docker.io`.
+We edit 
+`/var/snap/k8s/common/etc/containerd/hosts.d/docker.io/hosts.toml`
+and make sure it looks like this:
+
+##### HTTP registry
+
+In `/var/snap/k8s/common/etc/containerd/hosts.d/docker.io/hosts.toml`
+add the configuration:
+
+```
+[host."http://10.100.100.100:5000"]
+capabilities = ["pull", "resolve"]
+```
+
+##### HTTPS registry
+
+HTTPS requires that you additionally specify the registry CA certificate.
+Copy the certificate to
+`/var/snap/k8s/common/etc/containerd/hosts.d/docker.io/ca.crt`,
+
+Then add your config in
+`/var/snap/microk8s/current/args/certs.d/docker.io/hosts.toml`:
+
+```
+[host."https://10.100.100.100:5000"]
+capabilities = ["pull", "resolve"]
+ca = "/var/snap/k8s/common/etc/containerd/hosts.d/docker.io/ca.crt"
+```
+
+#### Container Runtime Option C: Side-load images
+
+Copy the `images.tar` file(s) to `/var/snap/k8s/common/images`
+on each cluster node.
+
+### Step 2: Bootstrap cluster
+
+Now, bootstrap the cluster and replace `MY-NODE-IP` with the IP of the node
+by running the command:
+
+```bash
+sudo k8s bootstrap --address MY-NODE-IP
+```
+
+```{note}
+Please skip the following section for one node deployments.
```

Comment:
I'd remove this. It is obvious for the user that the following only applies to multinode clusters.